app/assets/javascripts/media_player_wrapper/mejs4_plugin_playlist_items.es6

```diff
@@ -79,6 +80,27 @@ Object.assign(MediaElementPlayer.prototype, {
    * @type {Object}
    */
   playlistItemsObj: {
+    /**
+     * Add click listeners forplaylist item links
+     * @function addRelatedItemListeners
+     * @return {void}
+     */
+    addRelatedItemListeners() {
+      // Handle click on entire RelatedItems area
+      // Filter only <a> element clicks; disregard all others
+      if (this.$relatedItems) {
+        this.$relatedItems.on('click', (e) => {
+          if (e.target.nodeName === 'A') {
+            // find correct playlist item in sidebar and click it.
+            let playlist_id  = e.target.dataset.playlistId
+            let playlist_item_id = e.target.dataset.playlistItemId
+            let related = this.$sidePlaylist.find('a[data-playlist-id='+playlist_id+'][data-playlist-item-id='+playlist_item_id+']')
```

Comment:
Shouldn't `a[data-playlist-item-id]` be sufficient to uniquely identify the matching `<a>` given that the playlist-id should be shared by everything in the playlist?

Reply:
It's a matter of efficiency. Reducing the search scope.